src/content/reference/manifest.mdx

```diff
@@ -60,6 +60,20 @@ Each image supports the following fields:
 - `args`: add build arguments, which are environment variables accessible only during the build process. Build arguments with a value containing a `$` sign are resolved to the environment variable value on the machine okteto is running on.
 - `export_cache`: image tag for exported cache when build.
 - `cache_from`: list of images to import cache from.
+- `secrets`: list of secrets exposed to the build.
```

Comment:
```suggestion
- `secrets`: list of secrets' paths exposed to the build.
```

Reply:
How do these paths behave ? Are they relative to the current working directory or relative to the manifest?